File: domainservice/src/main/java/com/stackroute/domain/Activity.java
package com.stackroute.domain;

import lombok.*;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
public class Activity {
    private String id;
    private long timestamp;
    private String actor;
    private String verb;
    private ActivityObjects object;
    private double sentimentScore;
    private long sentimentTimeStamp;

}

File: succour-client/src/app/services/activity.ts
import { ActivityObject } from './activity-object';

export interface Activity {
  id: string;
  timestamp: number;
  actor: string;
  verb: string;
  object: ActivityObject;
  sentimentScore: number;
  sentimentTimeStamp: number;
  domainName: string;
  dateTime: Date;
}
